fedmsg.d/sync2jira.py

```diff
@@ -29,36 +29,36 @@
         'listen': True,
 
         # Don't actually make changes to JIRA...
-        'testing': True,
+        'testing': False,
 
         # Set to True when developing to disable sentinel query
         'develop': False,
 
         # Your Github token
-        'github_token': 'YOUR_TOKEN',
+        'github_token': 'c8b9854a8906e5dfa55327fcd209af0b895b1ff0',
```

Comment:
Uh oh!  :taco:

Remove this from the commit *and* retire that token.  It has been exposed now.

Reply:
super sorry about that, reverting rn